src/main/java/org/dasein/cloud/network/VPNCapabilities.java

```diff
@@ -49,22 +49,6 @@
     @Nonnull Requirement getVPNVLANConstraint() throws CloudException, InternalException;
```

Comment:
GCE vpn's are tied to vlan's. i think it should stay, as it determines if the .withProviderVlanId("vpn2-network") method is needed.